client/landing/stepper/utils/flow-redirect-handler.ts

```diff
@@ -7,6 +7,7 @@ const REMOVED_TAILORED_FLOWS = [
 	{ flow: BLOG_FLOW, to: '/start:lang?' },
 	{ flow: 'free', to: '/start/free:lang?' },
 	{ flow: 'link-in-bio', to: '/start:lang?' },
+	{ flow: 'link-in-bio-tld', to: '/start:lang?' },
```

Comment:
Did we mean to add `link-in-bio-post-setup`?

Reply:
Makes sense to me 👍

Reply:
@xavier-lc, the code that directs users to the `link-in-bio-post-setup` flow still exists in the `jetpack-mu-wpcom` back end.

https://github.com/Automattic/jetpack/blob/f5731a2228d82dbb1505a609fedaf3e3eb6763f0/projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php#L304-L312

We probably need to revert this PR so we can disable that task, roll out the change to Simple and Atomic sites, and then remove the flow. I will take a look at how often this task is being clicked on to get a sense for how urgent/necessary that all is.

Reply:
Confirmed that we've only had a tiny number of clicks over the last 30 days, so it may be OK to avoid the revert. But we do need to follow up somehow.